bip-taro.mediawiki

```diff
@@ -534,11 +534,14 @@ where:
 ** <code>split_commitment_proof</code>: is used to permit the spending of an asset UTXO created as a result of an asset split. When an asset is split, the non-change UTXO commits to the location of all other splits within an MS-SMT tree. When spending a change UTXO resulting from an asset_split, a normal <code>asset_witness</code> isn't required, instead the owner of the change asset UTXO must prove that it holds a valid split which was authorized by the main transfer transaction.
 *** Outputs with the same <code>split_commitment</code> are said to share a single <code>asset_witness</code> as such outputs are the result of a new asset split. Therefore we only need a single witness and the resulting merkle-sum asset tree to verify a transfer.
 ** <code>asset_witness</code>: is a serialized witness in an identical format as Bitcoin's Segwit witness field. This field can only be blank if <code>prev_asset_id</code> is blank.
+*** If <code>prev_asset_id</code> is all zeroes, the witness will be either
+**** empty, to indicate this asset does not support further issuance. In this case <code>asset_group_key</code> MUST NOT be present.
+**** on the format <code><asset_key_internal> <signature></code>, where <code>asset_key_internal</code> is the public key used to derive the <code>asset_group_key</code>, and <code>signature</code> is a 64-byte BIP 340 signature over the <code>asset_id</code> for <code>asset_group_key</code>. In this case <code>asset_group_key</code> MUST be present.
```

Comment:
"on the format" 

Seems like something in front of this got chopped off?

Reply:
It's a bullet list prepended with "either":

<img width="946" alt="Screenshot 2023-01-10 at 11 06 09" src="https://user-images.githubusercontent.com/3765732/211522107-1ddfbb9c-4f3a-4685-9b67-356f1636eae3.png">